Automattic/jetpack
# Jetpack AI Search: Release

This releases Jetpack AI Search to production

## Proposed changes:
* Switch Jetpack AI Search from a beta block to production block

## Does this pull request change what data or activity we track or use?
NO

## Testing instructions:
* Follow instructions from call for testing pdtkmj-1OZ-p2
* See that the block does not appear as beta anymore
